core/rlwe/metadata.go

```diff
@@ -146,20 +142,14 @@ func (m PlaintextMetaData) LogScale() float64 {
 func (m *PlaintextMetaData) Equal(other *PlaintextMetaData) (res bool) {
 	res = cmp.Equal(&m.Scale, &other.Scale)
 	res = res && m.IsBatched == other.IsBatched
+	res = res && m.IsBitReversed == other.IsBitReversed
 	res = res && m.LogDimensions == other.LogDimensions
 	return
 }
 
-// Equal returns true if two [CiphertextMetaData] structs are identical.
-func (m *CiphertextMetaData) Equal(other *CiphertextMetaData) (res bool) {
-	res = m.IsNTT == other.IsNTT
-	res = res && m.IsMontgomery == other.IsMontgomery
-	return
-}
-
 // BinarySize returns the size in bytes that the object once marshalled into a binary form.
 func (m PlaintextMetaData) BinarySize() int {
-	return 61 + m.Scale.BinarySize()
+	return 84 + m.Scale.BinarySize()
```

Comment:
what correspond 84?

Reply:
The metadata are encode in a fixed-size JSON (each field has a fixed size value), 84 is the number of bytes of this JSON struct.